makepanda/makewheel.py

```diff
@@ -0,0 +1,195 @@
+"""
+Generates a wheel (.whl) file from the output of makepanda.
+
+Since the wheel requires special linking, this will only work if compiled with
+the `--wheel` parameter.
+"""
+from __future__ import print_function, unicode_literals
+from distutils.util import get_platform as get_dist
+import json
+
+import sys
+import os
+from os.path import join
+import shutil
+import zipfile
+from makepandacore import ColorText
+
+
+def get_platform():
+    p = get_dist().replace('-', '_').replace('.', '_')
+    if "linux" in p:
+        print(ColorText("red", "WARNING:") +
+              " Linux-specific wheel files are not supported."
+              " We will generate this wheel as a generic package instead.")
+        return "any"
+    return p
+
+
+# Global filepaths
+LIB_SRC = join("built", "lib")  # TODO: A special wheel lib dir
+PANDA3D_SRC = join("built", "panda3d")  # TODO: Empty?
+PANDAC_SRC = join("built", "pandac")
+DIRECT_SRC = join("built", "direct")
+MODELS_SRC = join("built", "models")
+PLUGINS_SRC = join("built", "plugins")
+ETC_SRC = join("built", "etc")
+BIN_SRC = join("built", "bin")
+LICENSE_SRC = "LICENSE"
+README_SRC = "README.md"
+WHEEL_TARGET = join("built", "wheel")
+PANDA3D_TARGET = join(WHEEL_TARGET, "panda3d")
+PANDAC_TARGET = join(WHEEL_TARGET, "pandac")
+DIRECT_TARGET = join(WHEEL_TARGET, "direct")
+
+# Other global parameters
+PY_VERSION = "py3" if sys.version_info >= (3, 0) else "py2"
+PLATFORM_VERSION = get_platform()
+
+WHEEL_DATA = """Wheel-Version: 1.0
+Generator: bdist_wheel 1.0
+Root-Is-Purelib: false
+Tag: py2-none-any
+Tag: py3-none-any
```

Comment:
Should this tag not match what is indicated by the filename, ie. dependence on a specific CPython version?